packages/manager/apps/web/client/app/hosting/multisite/logs/hosting-multisite-logs.controller.js

```diff
@@ -35,22 +37,29 @@ angular.module('App').controller(
     /* eslint-disable no-param-reassign */
     generateLogHref(domain) {
       domain.logsLoading = true;
-      this.$q
-        .all({
-          ownLogs: this.HostingStatistics.getLogs(
-            this.$stateParams.productId,
-            domain.name,
-          ),
-          userLogsToken: this.Hosting.getUserLogsToken(
-            this.$stateParams.productId,
-            {
-              params: {
-                attachedDomain: domain.name,
-                remoteCheck: true,
+
+      return this.$http
+        .get(
+          `/hosting/web/${this.$stateParams.productId}/attachedDomain/${domain.name}`,
+        )
+        .then(({ data }) => data.ownLog)
+        .then((fqdn) =>
+          this.$q.all({
+            ownLogs: this.HostingStatistics.getLogs(
+              this.$stateParams.productId,
+              fqdn,
+            ),
+            userLogsToken: this.Hosting.getUserLogsToken(
+              this.$stateParams.productId,
+              {
+                params: {
+                  attachedDomain: domain.name,
+                  remoteCheck: true,
```

Comment:
Parameter `remoteCheck` must be removed.

Reply:
seen with issue reporter, parameter should be kept and we'll remove it in a future fix if it's still causing troubles